docs/src/main/asciidoc/ghpages.sh

```diff
@@ -60,7 +60,13 @@ function switch_to_tag() {
 # Build the docs if switch is on
 function build_docs_if_applicable() {
     if [[ "${BUILD}" == "yes" ]] ; then
-        ./mvnw clean install -P docs -pl docs -DskipTests -q
+        # Disable profile CI (inherited from spring-cloud-build/pom.xml) so
+        # process-asciidoc can be executed in GH action runner.
+        # Profile CI will be activated in GH action runner because
```

Comment:
I feel the profile name "CI" and its activation condition are bad. Is it clear why the profile "CI" is set up like that?

Reply:
CI profile was introduced based on this issue:
https://github.com/spring-cloud/spring-cloud-build/issues/239

> Don't access spring-doc-resources in CI builds[, because it] requires authentication to retrieve.

Reply:
That assumption (pull request builds = GitHub Actions) is wrong. Now I know it's outside our codebase.